csuntechlab/iSTART
# STR-47 -- FE: Login for admin users

# Description
Create admin route and logic; updated tests
  
# Testing
1. Log in as **`steve`**.
2. Ensure you are routed to admin panel.
3. Run **`yarn jest-unit`** and confirm all the test pass.
